tests/test_environs.py

```diff
@@ -215,6 +216,24 @@ def test_reading_proxied_variable_in_prefix_scope(self, set_env, env):
                 assert env.str("PASSWORD") == "nested-secret"
 
 
+class TestSpecificFileReading:
+    def test_read_env_recurse(self, env):
+        # Make sure to remove any 'CUSTOM_STRING' environment variable
+        if "CUSTOM_STRING" in os.environ:
```

Comment:
This probably isn't necessary. For consistency with the other similar tests, it should suffice to fail on the below sanity check.